book-library/index.html

```diff
@@ -75,9 +75,9 @@ <h1>Library</h1>
         <tr>
           <th>Title</th>
           <th>Author</th>
-          <th>Number of Pages</th>
-          <th>Read</th>
-          <th></th>
+          <th>Pages</th>
+          <th>check</th>
+          <th>display</th>
```

Comment:
These table headings do not need to be changed.  They should be "Number of pages" (in the book) and "Read" (a column to indicate whether or not I have read the book.  In the Javascript these are called "pages" and "check", but those are not the correct description of the columns to present to the user

Reply:
I have changed it from (check to Number of Pages)  and  added one more  <th> Read</th>  under Display .